server/README.md

```diff
@@ -0,0 +1,181 @@
+# Attestation Server
+
+Attestation Server is a user space application for attestation purpose. 
+It receives and verifies the TEE's attestation [Evidence](https://github.com/confidential-containers/kbs/blob/main/docs/kbs_attestation_protocol.md#attestation) and responses the corresponding [Attestation Results](https://github.com/confidential-containers/attestation-service/issues/1) to ensure the Evidence's generation environment is a real TEE environment and it's Trusted Computing Base (TCB) status are as expected.
+
+Attestation Server supports the following types of TEE:
+- SGX
+- TDX
+- SEVSNP
+- SAMPLE: Dummy TEE which is used to test/demo the Server's functionalities.
+
+Attestation Server depends on [Open Policy Agent (OPA)](https://www.openpolicyagent.org/docs/latest/) to evaluate the Evidence's TCB status during attestation. Each supported TEE's OPA `Policy(.rego)` and `Reference Data(.json)` can be customized in order to evaluate the TCB status precisely.
+Note: Please refer [Policy Language](https://www.openpolicyagent.org/docs/latest/policy-language/) for more about `.rego` syntax information.
+
+## API
+
+Attestation Server provides two groups of GRPC endpoints:
+- attestation: Attest the received Evidence which is defined in [protobuf](https://github.com/confidential-containers/attestation-service/server/proto/attestation.proto).
+- management: Customize the Server's configuration which is defined in [protobuf](https://github.com/confidential-containers/attestation-service/server/proto/management.proto).
+
+### Attestation
+
+The `attestation` group only includes one `Attestation` GRPC endpoint currently. It receives the attestation [Evidence](https://github.com/confidential-containers/kbs/blob/main/docs/kbs_attestation_protocol.md#attestation), executes attestation procedures, and responses the [Attestation Results](https://github.com/confidential-containers/attestation-service/issues/1).
+
+`Attestation` message:
+```PROTO
+message AttestationRequest {
+    // Attestation evidence: https://github.com/confidential-containers/kbs/blob/main/docs/kbs_attestation_protocol.md#attestatio
+    bytes evidence = 1;
+    // Optional: Designate the user id. It should be keeped as "None" currently.
+    optional string user = 2;
+}
+```
+
+### Management
+
+It's mainly used to customize Attestation Server's configurations:
+- Customize each type of TEE's `Policy(.rego)` and `Reference Data(.json)` in order to evaluate it's TCB status precisely.
+    - GetPolicy
+    - GetReferenceData
+    - SetPolicy
+    - SetReferenceData
+    - ResetPolicy
+    - ResetReferenceData
+- Test/Evaluate the local `Policy(.rego)` or `Reference Data(.json)` with remote Attestation Server's OPA engine.
+    - TestOpa
```

Comment:
Should we also have an endpoint to get logging information from the AS or is that better left to the KBS?

Reply:
Acutally I didn't consider to enable the logging information's storage for AS now. Currently, AS's debug log can be enabled by `RUST_LOG=debug attestation-server xxxxx`. Maybe we can add a better logging mechanism and extend the endpoint set to add a `get_logging()` endpoint in other PR in future.

Reply:
Yes, I agree with you that maybe it's better to let KBS keep this recording. For AS, the Attesattion Results includes all these evaluation information, so I think it should be enough to let the results act as logging also.